Business Rules/RITM state change/Related_task_state_update.js

```diff
@@ -0,0 +1,16 @@
+//create business rule on sc_req_item table and add condtion 
+
+(function executeRule(current, previous /*null when async*/ ) {
+
+    var gr = new GlideRecord("sc_task");
+    gr.addQuery('request_item', current.sys_id);
+    gr.orderBy('order');
+    gr.setLimit(1);
```

Comment:
What if more than one sc_task is under RITM?

Reply:
removed setLimit(1) so that it will update all the sc_task related to RITM